zmon-controller-app/src/main/java/org/zalando/zmon/service/impl/ZMonServiceImpl.java

```diff
@@ -639,9 +641,20 @@ private ExecutionStatus buildStatus(int alertsActive, final Map<String, Response
             final String invocation = invocations.get(lastUpdate.getKey()).get();
             builder.addWorker(lastUpdate.getKey(), (long) ts, invocation == null ? 0 : Long.valueOf(invocation));
         }
-
         builder.withWorkersActive(workersActive);
 
+        List<String> entities = entitySProc.getEntityById("zmon-service-level-config");
+        log.info(entities.get(0));
```

Comment:
1) this will fail in runtime if `entities` is empty
2) not sure if `INFO` is a right log level for this case
3) please always add some context to the log line, e.g. `"service level config for execution status: " + entities.get(0)`. Otherwise you can have sometimes empty JSON object logged without any explanation what does it mean